src/templates/post.html

```diff
@@ -23,52 +27,91 @@
+			</span>
+			{% if post.since4pass %}<span class="n-pu" title="Pass user since {{post.since4pass}}."></span>{% endif %}
+			{% if post.id %}<span class="posteruid id_{{post.id}}">(ID: <a title="." href="#">{{post.id}}</a>) </span>{% endif %}
+			{% if post.country %}<span title="{{post.country}}{{post.country_name}}" class="flag flag-{{post.country|lower}}"></span>{% endif %}
+			{% if post.troll_country %}<span title="{{post.country_name}}" class="flag-pol2 flag-{{post.troll_country|lower}}"></span>{% endif %}
             <span class="dateTime" data-utc="{{post.time}}">{{post.now}}</span>
             <span class="postNum desktop">
-                <a href="#p{{post.no}}" title="Link to this post">No.</a>
-                <a href="javascript:quote('{{post.no}}');" title="Reply to this post">{{post.no}}</a>
+				{% if search_result and post.resto != 0 %}
+					<a href="/{{post.board_shortname}}/thread/{{post.resto}}#p{{post.no}}">No.{{post.no}}</a>
+				{% elif search_result and post.resto == 0 %}
+					<a href="/{{post.board_shortname}}/thread/{{post.no}}#p{{post.no}}">No.{{post.no}}</a>
+				{% else %}
+					<a href="#p{{post.no}}" title="Link to this post">No.</a><a href="javascript:quote('{{post.no}}');" title="Reply to this post">{{post.no}}</a>
+				{% endif %}
+				{% if post.resto == 0 %}
+					{% if post.sticky %}
+						<img src="/static/images/sticky.gif" alt="Sticky" title="Sticky" class="stickyIcon retina">
+					{% endif %}
+					{% if post.closed %}
+						<img src="/static/images/closed.gif" alt="Closed" title="Closed" class="closedIcon retina">
+					{% endif %}
+				{% endif %}
             </span>
             <a href="#" class="postMenuBtn" title="Post menu" data-cmd="post-menu">▶</a>
-
-            {% if search_result and post.resto != 0 %}
-                <a href="/{{post.board_shortname}}/thread/{{post.resto}}#p{{post.no}}">View</a>
+			{% if search_result and post.resto != 0 %}
+                [<a href="/{{post.board_shortname}}/thread/{{post.resto}}#p{{post.no}}" target="_blank">View</a>]
             {% endif %}
             {% if search_result and post.resto == 0 %}
-                <a href="/{{post.board_shortname}}/thread/{{post.no}}#p{{post.no}}">View</a>
+                [<a href="/{{post.board_shortname}}/thread/{{post.no}}" target="_blank">View</a>]
             {% endif %}
-
             <div id="bl_{{post.no}}" class="backlink">
-                {% if quotelinks %}
+			{% if quotelinks %}
                 {% for quotelink in quotelinks[post.no|string] %}
                     <span class="quotelink">
-                        <a href="#p{{ quotelink }}" class="quotelink" data-board_shortname="{{post.board_shortname}}">&gt;&gt;{{ quotelink }}</a>
```

Comment:
`data-board_shortname` needs to remain here since it's use in `index.js` to build URLs for fetch requests for posts from different boards.